src/screens/NewAnalytics/PaymentAnalytics/PaymentsProcessed/PaymentsProcessedTypes.res

```diff
@@ -10,3 +10,5 @@ type paymentsProcessedCols =
   | Amount
   | Currency
   | TimeBucket
+
+type queryData = [#Count | #Amount | #TimeBucket]
```

Comment:
Amount, Currency, TimeBucket are already defined as a variant in the top, uc an use them only or remove that and have only polyvariant

Reply:
okay